apps/arena/lib/arena/game_launcher.ex

```diff
@@ -86,11 +90,31 @@ defmodule Arena.GameLauncher do
     batch_start_at
   end
 
-  defp get_bot_clients(missing_clients) do
-    Enum.map(1..missing_clients, fn _ ->
+  defp get_bot_clients(missing_clients, game_config) do
+    bot_names = [
+      "TheBlackSwordman",
+      "SlashJava",
+      "SteelBallRun",
+      "Bot h4ck",
```

Comment:
Appreciate the meme but let's change this so it doesn't look like a bug 😂 Same with "Bot Name"

Reply:
done! 😭